sous-chef/tests/unit_tests/menu/create_menu/conftest.py

```diff
@@ -65,12 +67,18 @@ def create_all_menu_row(
         item_type: str = "recipe",
         eat_factor: float = 1.0,
         # gsheets has "", whereas read_csv defaults to np.nans
-        eat_unit: str = "",
+        eat_unit: Union[Unit, str] = "",
```

Comment:
need union here?

Reply:
No, I was able to remove it. I think it was only relied on in a few minor test cases that I modified.